drivers/telescope/lx200ap_experimental.cpp

```diff
@@ -397,35 +428,69 @@ bool LX200AstroPhysicsExperimental::initMount()
 
     return true;
 }
+#endif
 
 /**************************************************************************************
 **
 ***************************************************************************************/
 bool LX200AstroPhysicsExperimental::ISNewNumber(const char *dev, const char *name, double values[], char *names[], int n)
 {
+    LOG_DEBUG("ISNewNumber entry");
     if (strcmp(getDeviceName(), dev))
         return false;
 
-    if (!strcmp(name, MeridianDelayNP.name))
+    if (!strcmp(name, APUTCOffsetNP.name))
     {
-        if (IUUpdateNumber(&MeridianDelayNP, values, names, n) < 0)
+        if (IUUpdateNumber(&APUTCOffsetNP, values, names, n) < 0)
             return false;
 
         float mdelay;
         int err;
 
-        mdelay = MeridianDelayN[0].value;
+        mdelay = APUTCOffsetN[0].value;
 
-        LOGF_INFO("lx200ap_experimental: meridian delay request = %f", mdelay);
-
-        if (!isSimulation() && (err = setAPMeridianDelay(PortFD, mdelay) < 0))
+        if (!isSimulation() && (err = setAPUTCOffset(PortFD, mdelay) < 0))
         {
-            LOGF_ERROR("lx200ap_experimental: Error setting meridian delay (%d).", err);
+            LOGF_ERROR("lx200ap_experimental: Error setting UTC offset (%d).", err);
             return false;
         }
 
-        MeridianDelayNP.s = IPS_OK;
-        IDSetNumber(&MeridianDelayNP, nullptr);
+        APUTCOffsetNP.s  = IPS_OK;
+        IDSetNumber(&APUTCOffsetNP, nullptr);
+
+        return true;
+    }
+    if (!strcmp(name, HourangleCoordsNP.name))
+    {
+        HourangleCoordsNP.s = IPS_BUSY;
+        IDSetNumber(&HourangleCoordsNP, nullptr);
```

Comment:
Why do you need to explicitly set it to IPS_BUSY and send it over to client here?

Reply:
I wanted the user to realize that HA is updated.